src/main/java/com/heeverse/ticket/service/TicketService.java

```diff
@@ -128,8 +137,16 @@ public List<TicketRemainsResponseDto> getTicketRemains(long concertSeq) {
                 .collect(Collectors.toList());
     }
 
-    public void rollbackTicketOrderSeq(List<Long> ticketSeqList) {
-        int rollbackCount = ticketMapper.rollbackTicketOrderSeq(ticketSeqList);
-        log.info("티켓 예매 rollback success count : {}", rollbackCount);
+    @Transactional(propagation = Propagation.MANDATORY, isolation = Isolation.READ_COMMITTED, noRollbackFor = AlreadyBookedTicketException.class)
+    public void checkBookedTicket(TicketOrderRequestDto dto, List<Long> reqTicketSeqList) {
+        List<Ticket> availableTicketList = ticketMapper.findTicketsByTicketSeqList(reqTicketSeqList)
+                .stream()
+                .filter(d -> ObjectUtils.isEmpty(d.getOrderSeq()))
```

Comment:
`ObjectUtils.isEmpty(d.getOrderSeq())`이 무준을 아직 하지 않은 티켓을 나타내는 비즈니스 로직이라면, 이런 형식으로 메서드 네이밍을 붙여서 가독성과 어떤 로직인지를 명시해줄 수 있습니다.
```suggestion
                .filter(d -> d.isNotOrdered())
```

```suggestion
                .filter(Ticket::isNotOrdered)
```